app/lib/konnectors.js

```diff
@@ -89,7 +89,7 @@ export function run (cozy, slug, accountId, folderId, timeout = 120 * 1000) {
       }
     })
   })
-  .then(job => waitForJobFinished(cozy, job, timeout))
+  //.then(job => waitForJobFinished(cozy, job, timeout))
```

Comment:
The linting will failed because `waitForJobFinished` will never be called.

What about dealing right now with an error from the stack ? It may happen later in production and we should be ready to handle it. 

I suggest to let the error be thrown and the whole process fail. When the stack will be ready, it will work without adding any commit from our side. And for development purpose we are still able to comment it.

What do you think ?